apps/tutorial-servo-can-control-steven/main.cpp

```diff
@@ -0,0 +1,24 @@
+/* mbed Microcontroller Library
+ * Copyright (c) 2018 ARM Limited
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "CANMsg.h"
+#include "TutorialServo.h"
+#include "mbed.h"
+
+// Declare global variables
+CAN can(CAN1_RX, CAN1_TX);
+CANMsg msg;
+TutorialServo servo(PA_1);
+unit8_t convertedMessage = 0;
```

Comment:
typo here. also i think this a float in the training doc. you should always try to compile your code on ur machine btw